Funcky.Test/OptionTest.cs

```diff
@@ -464,6 +464,21 @@ public void InspectDoesNotExecuteSideEffectWhenOptionIsNone()
             Assert.Equal(option, inspectedOption);
         }
 
+        [Fact]
+        public void GivenAFunctionWithADefaultOptionParameterGivesANoneWhenNoValueGiven()
+        {
+            MethodWithDefaultOptionParameter()
+                .Match(none: () => Assert.True(true), some: s => Assert.True(false));
+
+            MethodWithDefaultOptionParameter(Option.Some("value"))
```

Comment:
Use `FunctionalAssert.IsSome("value", s)`

Reply:
fixed in 9d88d4fb5c71997a25a21b6c8c563f5dcd354b70